src/tests/ftest/erasurecode/rebuild_disabled_single.yaml

```diff
@@ -7,7 +7,7 @@ hosts:
     10_server:
       test_servers: server-[1-5]
   test_clients: 1
-timeout: 400
+timeout: 1900
```

Comment:
Why is this 400->1900 needed?

Reply:
Tests failed randomly when test time 1500. 
1900 is realistic.

Reply:
An extra 25 minutes is a lot. I know this test is currently failing, but how could it ever have been passing with only 400s? it seems there might be some other bug or core change here?